optuna/importance/__init__.py

```diff
@@ -2,14 +2,18 @@
 from typing import Dict
 from typing import List
 from typing import Optional
+from typing import TYPE_CHECKING
```

Comment:
Let's try avoiding having to rely on `TYPE_CHECKING` to be consistent. (I know you're working on it but leaving a comment for the record 🙇 )